app/locales/fr.json

```diff
@@ -17,171 +17,171 @@
+    "miniBreaks": "Petites pauses :",
+    "postponableFor": "Peut être reporté de :",
+    "postponeFor": "Reporter de :",
+    "maxPostpones": "Nombre de reports maximum :",
+    "longBreaks": "Grandes pauses :",
+    "pauseUntilMoring": "Suspendre jusqu'au matin :",
+    "pauseUntil": "Suspendre jusqu'à :",
+    "pauseUntilMorningInfo": "Spécifiez jusqu'à quelle heure « Suspendre jusqu'au matin » doit suspendre les pauses.",
+    "welcomeWindow": "Écran de bienvenue :",
+    "showWelcomeWindow": "Afficher l'écran de bienvenue au prochain démarrage",
+    "sounds": "Sons :",
+    "volume": "Volume :",
+    "miniBreakStartSound": "Jouer le son de fin de pause également au début d'une petite pause",
+    "longBreakStartSound": "Jouer le son de fin de pause également au début d'une grande pause",
+    "naturalBreaks": "Pauses naturelles :",
+    "naturalBreaksInfo": "Si la surveillance des périodes d'inactivité est activée dans les paramètres principaux, vous pouvez préciser à partir de quelle durée le décompte doit être suspendu.",
+    "pauseAfter": "Suspendre après :",
+    "appearance": "Apparence :",
+    "light": "Clair",
+    "dark": "Sombre",
+    "system": "Système"
   },
   "utils": {
     "none": "{{count}}",
     "none_plural": "{{count}}",
-    "percent": "{{count}} percent",
-    "percent_plural": "{{count}} percent",
-    "oclock": "{{count}} o'clock",
-    "oclock_plural": "{{count}} o'clock",
-    "seconds": "{{count}} second",
-    "seconds_plural": "{{count}} seconds",
+    "percent": "{{count}} pourcent",
+    "percent_plural": "{{count}} pourcent",
+    "oclock": "{{count}} heure",
+    "oclock_plural": "{{count}} heures",
+    "seconds": "{{count}} seconde",
+    "seconds_plural": "{{count}} secondes",
     "minutes": "{{count}} minute",
     "minutes_plural": "{{count}} minutes",
-    "hours": "{{count}} hour",
-    "hours_plural": "{{count}} hours",
-    "secondsRemaining": "$t(utils.seconds, {'count': {{seconds}} }) remaining",
-    "aboutMinutesRemaining": "About $t(utils.minutes, {'count': {{minutes}} }) remaining",
-    "aboutHoursRemaining": "About $t(utils.hours, {'count': {{hours}} }) remaining",
-    "aboutHoursMinutesRemaining": "About $t(utils.hours, {'count': {{hours}} }) $t(utils.minutes, {'count': {{minutes}} }) remaining",
-    "inSeconds": "in $t(utils.seconds, {'count': {{seconds}} })",
-    "inAboutMinutes": "in about $t(utils.minutes, {'count': {{minutes}} })",
-    "inAboutHours": "in about $t(utils.hours, {'count': {{hours}} })",
-    "inAboutHoursMinutes": "in about $t(utils.hours, {'count': {{hours}} }) $t(utils.minutes, {'count': {{minutes}} })"
+    "hours": "{{count}} heure",
+    "hours_plural": "{{count}} heures",
+    "secondsRemaining": "$t(utils.seconds, {'count': {{seconds}} }) restantes",
```

Comment:
@hovancik We could use a plural version of this one and the variants below too.

Reply:
do you need plural in French there?

In English, also in Slovak it's the same word. 

`secondsRemaining` is 1-60 seconds
`aboutMinutesRemaining` is 2-60 minutes
`aboutHoursRemaining` is 2 - xxx hours
`aboutHoursMinutesRemaining` is 2+hours 1-59 minutes

Reply:
The original string was `"$t(utils.seconds, {'count': {{seconds}} }) remaining"`. In this string, "remaining" has a plural form in French:
- 1 seconde restante
- 2 secondes restante**s**

So yes, a plural form here would be helpful :slightly_smiling_face:

Reply:
I will have to think, because I don't see way to do it right now. (With my nested form)

Reply:
Well that's not a big deal, we can still sort this out later :)

Reply:
Ok :+1: I'll have to check if some libraries exist for this